sbt-plugin-test/build.sbt

```diff
@@ -98,6 +98,19 @@ lazy val multiTest = crossProject.
 lazy val multiTestJS = multiTest.js
 lazy val multiTestJVM = multiTest.jvm
 
+lazy val jUnitTest = project.settings(baseSettings: _*).
+  enablePlugins(ScalaJSPlugin).
+  enablePlugins(ScalaJSJUnitPlugin).
```

Comment:
I think we should put the JUnit test within `multiTest` instead, with cross-compiling JUnit tests, and JUnit tests specific to the JVM and JS.